packages/okta-angular/test/e2e/harness/e2e/app.e2e-spec.ts

```diff
@@ -174,8 +174,8 @@ describe('Angular + Okta App', () => {
 
       sessionTokenSignInPage.waitUntilVisible();
       sessionTokenSignInPage.signIn({
-        username: process.env.USERNAME,
-        password: process.env.PASSWORD
+        username: environment.USERNAME,
```

Comment:
Not sure why this would make the test flaky as opposed to "won't work", but clearly this is better.

Reply:
what I saw from bacon is latest PDVoidc are all green, but this one failed in my local, as you said, would be good to fix it.